ci_framework/roles/edpm_deploy/tasks/main.yml

```diff
@@ -14,6 +14,14 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Set EDPM container registry related vars
+  tags:
+    - always
+  ansible.builtin.set_fact:
```

Comment:
If these var is used only in a single place why creating a fact for a single use? Can be perfectly placed in https://github.com/openstack-k8s-operators/ci-framework/blob/7c493a6d767b51339bc59fc2168b40b0f64e7ee3/ci_framework/roles/edpm_deploy/tasks/main.yml#L25-L36

Reply:
Done